src/acl/Asn.cc

```diff
@@ -595,7 +595,7 @@ ACLDestinationASNStrategy::match (ACLData<MatchType> * &data, ACLFilledChecklist
     const ipcache_addrs *ia = ipcache_gethostbyname(checklist->request->url.host(), IP_LOOKUP_IF_MISS);
 
     if (ia) {
-        for (const auto ip: ia->goodAndBad()) {
+        for (const auto &ip: ia->goodAndBad()) {
```

Comment:
Could you please let me know what warning did you get for this line. I agree that we should use a reference here, if possible, but I am curious what the compiler was warning us about in this a similar places.

Reply:
Never mind. I found the `-Wrange-loop-construct` answer in your branch commit and am resolving this change request. Please note that we normally _squash_ all branch commits, so, if there is useful information in them, we have to copy it to the PR description. This also helps avoid fishing for info in branch commits.